packages/cgb-scripts/template/src/init.php

```diff
@@ -47,7 +47,7 @@ function <% blockNamePHPLower %>_cgb_editor_assets() {
 	wp_enqueue_script(
 		'<% blockNamePHPLower %>-cgb-block-js', // Handle.
 		plugins_url( '/dist/blocks.build.js', dirname( __FILE__ ) ), // Block.build.js: We register the block here. Built with Webpack.
-		array( 'wp-blocks', 'wp-i18n', 'wp-element' ), // Dependencies, defined above.
```

Comment:
These dependencies are required:

1. `wp-blocks` handles block registration & various other block API's
2. `wp-i18n` handles translations 
3. `wp-element` react createElement abstraction

Reply:
@ajitbohra Can you fix this PR?

Reply:
@ahmadawais sure

Reply:
Any updates?! 🤔